include/nds/interrupts.h

```diff
@@ -253,6 +253,9 @@ void irqDisableAUX(u32 irq);
 ///     This doesn't actually use a software interrupt, it's a custom function
 ///     implemented in libnds.
 void swiIntrWait(u32 waitForSet, uint32_t flags);
+#ifdef ARM7
```

Comment:
This needs to be documented (well, and the other ARM7 functions of this file, they are ignored by doxygen because the description only affects one function, not two).

Reply:
https://github.com/blocksds/libnds/commit/78f1a5e760904a86ebecf928e222eee62a5a01d0